### PR TITLE
Fixes #34964 - update readme with installer instructions

### PR DESCRIPTION
**Pull request by jeremylenz**
Now that the installer takes care of the broker configuration, the README can be simplified.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,19 +12,19 @@ After a successful run, it is safe to remove katello-agent from the client.
...
-
-* Message broker such as `mosquitto` running
-* smart_proxy_remote_execution_ssh configured as follows
-```yaml
-# /etc/foreman-proxy/settings.d/remote_execution_ssh.yml
-:mqtt_broker: localhost
-:mqtt_port: 1883
-:mode: pull-mqtt
+* Foreman version 3.3 or later
+* foreman_proxy_remote_execution v ?.?? or later
```
Comment (jeremylenz) [inline README.md]: @adamruzicka help pls?

also, am I forgetting any other prerequisites?

Comment (adamruzicka) [reply README.md]: REX 7.0.0 or higher on Foreman's side
smart_proxy_remote_execution_ssh 0.7.0 or higher on proxy side

**Comment by jeremylenz** [comment]
@adamruzicka updated

**Review by adamruzicka — APPROVED** [review]
LGTM

**Comment by adamruzicka** [comment]
Thank you @jeremylenz !

**Pull request merged**